rocon 1.2.4 causes compile error

This code causes compile error using rocon v1.2.4.
With rocon v1.2.3, it does not happen compile error.

```ts
// Copy from https://rocon.uhyohyo.net/tutorial/navigation
import Rocon, { useNavigate } from "rocon/react";
import React from "react";

const toplevelRoutes = Rocon.Path()
  .any("id", {
    action: ({ id }) => <p>Hello, {id}</p>
  });

const navigate = useNavigate();
navigate(toplevelRoutes.anyRoute, { id: "uhyo" });
```

```console
❯ yarn run tsc --noEmit
yarn run v1.22.17
$ /Users/user/work/js/example-rocon/node_modules/.bin/tsc --noEmit
index.tsx:10:35 - error TS2554: Expected 1 arguments, but got 2.

10 navigate(toplevelRoutes.anyRoute, { id: "uhyo" });
                                     ~~~~~~~~~~~~~~


Found 1 error.

error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

https://gist.github.com/mtgto/36d02086138f37ed368250dfe47802cf

--- a/src/builder/PathRouteBuilder/PathRouteBuilder.test.ts
+++ b/src/builder/PathRouteBuilder/PathRouteBuilder.test.ts
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
 import { PathRouteBuilder } from ".";
 import { RoutePathResolver } from "../RoutePathResolver";
 import { PathRouteRecord } from "../RouteRecord";
@@ -175,6 +176,15 @@ describe("PathRouteBuilder", () => {
       });
       expect(res.anyRoute.action({ id: "wow" })).toEqual("id is wow");
     });
+    it("any route requires Match object", () => {
+      const res = PathRouteBuilder.init<string>().any("id", {
+        action: ({ id }) => `id is ${id.slice(0, 8)}`,
+      });
+      expect(() => {
+        // @ts-expect-error
+        res.anyRoute.action({});
+      }).toThrow();
+    });
     it("sub route of any", () => {
       const res = PathRouteBuilder.init<string>().any("id", {
         action: ({ id }) => `id is ${id.slice(0, 8)}`,
@@ -247,7 +257,7 @@ describe("PathRouteBuilder", () => {
           action: ({ id }) => `id is ${id.slice(0, 8)}`,
         });
       expect(res.exactRoute.action({})).toBe("I am root");
-    })
+    });
     describe("resolve", () => {
       it("resolve root", () => {
         const toplevel = PathRouteBuilder.init()

--- a/src/builder/PathRouteBuilder/index.ts
+++ b/src/builder/PathRouteBuilder/index.ts
@@ -298,7 +298,7 @@ export class PathRouteBuilder<
         routeDefinition?.action
       ),
     };
-    result.#exactRoute = this.#exactRoute
+    result.#exactRoute = this.#exactRoute;
     return result;
   }
 
@@ -354,11 +354,11 @@ export class PathRouteBuilder<
   /**
    * Route record of the any route.
    */
-  get anyRoute(): AnyRouteType<ActionResult, AnyFlag, Match> {
+  get anyRoute(): AnyRouteType<ActionResult, AnyFlag, AnyMatch> {
     return this.#wildcardRoute?.route as AnyRouteType<
       ActionResult,
       AnyFlag,
-      Match
+      AnyMatch
     >;
   }
 

